ONSdigital/alpha-eq-survey-runner
# Digeq 159 negative numbers

**What**
Fixes bug DIGEQ-159 and allows a user to enter a negative number in a numeric field. Also refactoring the validators into their own directory and adding a unit test for numeric. 

**Note**
We probably should add test classes for all validators

**How to test**
1. Create a questionnaire with a numeric question
2. Add no additional validation rules
3. Check that you can enter negative numbers

**Who can review**
Anyone apart from @warren-methods
